vcd/resource_vcd_org_vdc_test.go

```diff
@@ -202,12 +202,12 @@ func testAccCheckVcdVdcExists(name string, vdc *govcd.Vdc) resource.TestCheckFun
 			return fmt.Errorf(errorRetrievingOrg, testConfig.VCD.Org+" and error: "+err.Error())
 		}
 
-		newVdc, err := adminOrg.GetVdcByName(rs.Primary.Attributes["name"])
+		newVdc, err := adminOrg.GetVDCByName(rs.Primary.Attributes["name"], false)
 		if err != nil {
-			return fmt.Errorf("vdc %s does not exist (%#v)", rs.Primary.Attributes["name"], newVdc)
+			return fmt.Errorf("vdc %s does not exist (%s)", rs.Primary.Attributes["name"], err)
 		}
 
-		vdc = &newVdc
+		vdc = newVdc
```

Comment:
We're not returning it. Is it needed?

Reply:
No. see note about the similar variable for catalog